src/pythonjsonlogger/jsonlogger.py

```diff
@@ -179,7 +179,7 @@ def jsonify_log_record(self, log_record):
 
     def serialize_log_record(self, log_record):
         """Returns the final representation of the log record."""
-        return "%s%s" % (self.prefix, self.jsonify_log_record(log_record))
+        return "%s%s" % (self.prefix, self.jsonify_log_record(log_record).replace('\\n', '\n'))
```

Comment:
I think this will break the parsing of the log message. It won't be valid json any more.
You can't have literal `\n` in a json string

Reply:
Producing valid JSON is key here, and in addition to the above point, this PR changes the behaviour for all users regardless of whether they actually want that. But producing invalid JSON is not something that we should do anyway.